kubernetes/cache/cache.go

```diff
@@ -82,6 +82,8 @@ type (
 	}
 )
 
+var stopCacheChan chan bool
```

Comment:
This shouldn't be a global var. It's state that should be scoped to the `kialiCacheImpl` since nothing outside of that needs to use it. Can you make this a field on the struct?

Reply:
Updated.